index.html

```diff
@@ -50,6 +50,141 @@ <h1>I’m Eugene. <br>
+                            <button class="btn-action">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-container">
+                    <div class="card-container">
+                        <div class="card">
+                            <img src="images/Snapshoot Portfolio (6).svg" alt="">
+                        </div>
+                        <div class="left-block">
+                            <h2>Tonic</h2>
+                            <ul class="position">
+                                <li>CANOPY<i class="fa fa-circle"></i></li>
+                                <li>Back End Dev<i class="fa fa-circle"></i></li>
+                                <li>2015</li>
+                            </ul>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                            <ul class="programming-lang">
+                                <li><a href="#">HTML</a></li>
+                                <li><a href="#">CSS</a></li>
+                                <li><a href="#">Javascript</a></li>
+                            </ul>
+                            <button class="btn-action">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-container">
+                    <div class="card-container">
+                        <div class="card">
+                            <img src="images/Snapshoot Portfolio (7).svg" alt="">
+                        </div>
+                        <div class="left-block">
+                            <h2>Tonic</h2>
+                            <ul class="position">
+                                <li>CANOPY<i class="fa fa-circle"></i></li>
+                                <li>Back End Dev<i class="fa fa-circle"></i></li>
+                                <li>2015</li>
+                            </ul>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                            <ul class="programming-lang">
+                                <li><a href="#">HTML</a></li>
+                                <li><a href="#">CSS</a></li>
+                                <li><a href="#">Javascript</a></li>
+                            </ul>
+                            <button class="btn-action">See project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        <div class="about-myself">
+           <div class="about-container">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a href="#" class="connect">
```

Comment:
- [ ] Kindly use a header tag `hx`

Reply:
- [ ] As requested by the previous reviewer, kindly use a `hx` tag such as `h2`, `h3`, and `h4` for the `About Myself` element, etc. instead of the `h1` tag. This is because in order to have proper page structure, the page needs to only have one `h1` tag

Reply:
- [ ] As requested here kindly wrap line 138 in an `h2` tag instead of `h1` tag for semantic purposes